### PR TITLE
cookbook for a temperature field from an ascii file

### PR DESCRIPTION
**Pull request by Francyrad**
Pull Request Checklist. Please read and check each box with an X. Delete any part not applicable. Ask on the [forum](https://community.geodynamics.org/c/aspect) if you need help with any step.

*Describe what you did in this PR and why you did it.*
I added a cookbook that correctly explain how to build an ASCII file for ASPECT from zero.
I tried to do my best, is the first time that I use Latex, it's all ok in an online Latex viewer, so let me know if there is something that is not ok...
### Before your first pull request:

* [X ] I have read the guidelines in our [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md) document.

### For all pull requests:

* [X] I have followed the [instructions for indenting my code](../blob/main/CONTRIBUTING.md#making-aspect-better).

### For new features/models or changes of existing features:

* [ ] I have tested my new feature locally to ensure it is correct.
* [ ] I have [created a testcase](https://aspect-documentation.readthedocs.io/en/latest/user/extending/testing/writing-tests.html) for the new feature/benchmark in the [tests/](../blob/main/tests/) directory.
* [ ] I have added a changelog entry in the [doc/modules/changes](../blob/main/doc/modules/changes) directory that will inform other users of my change.

### CODE REVIEW

File: cookbooks/ascii_perturbation/ascii_perturbation.prm [hunk 1]
```diff
@@ -0,0 +1,179 @@
+
```
Comment (naliboff) [inline cookbooks/ascii_perturbation/ascii_perturbation.prm:1]: Can you add a brief description (2-4 sentences) of the benchmark at the top of the PRM file?

File: cookbooks/ascii_perturbation/documentation/tutorial_a.docx
Comment (naliboff) [inline cookbooks/ascii_perturbation/documentation/tutorial_a.docx:1]: Can you remove this file from the PR?

File: cookbooks/ascii_perturbation/documentation/tutorial_ascii.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+The final result will be as following:
+
+```
+<img src="ascii_temperature.png" style="width:48.0%" />
+
+Temperature distribution in ASCII format.
+```
+
+
+Now, let's write a Python script to utilize the function and export it in ASCII format
```
Comment (naliboff) [inline cookbooks/ascii_perturbation/documentation/tutorial_ascii.md:87]: Thank you for including the python code! However, as you have already provided it as a separate .py file in the cookbook folder, I think just a brief overview of what the script does would be fine in the cookbook folder.

File: cookbooks/ascii_perturbation/documentation/tutorial_ascii.tex
Comment (naliboff) [inline cookbooks/ascii_perturbation/documentation/tutorial_ascii.tex:1]: Similar to the comment above, can you remove this file from the pull request?

File: cookbooks/ascii_perturbation/documentation/perturbation.txt
Comment (naliboff) [inline cookbooks/ascii_perturbation/documentation/perturbation.txt:1]: Is this file redundant with the file cookbooks/ascii_perturbation/perturbation_ascii.txt? If yes, can you remove it?

**Review by naliboff — CHANGES_REQUESTED** [review]
@Francyrad - Thank you for this contribution! I have a few initial first-order comments and suggestions, and after that is done I will do a more detailed review of the PRM file, manual addition, and python script. Please let me know if you have any questions about the git procedures for removing files!

File: cookbooks/ascii_perturbation/documentation/perturbation.txt
Comment (Francyrad) [reply cookbooks/ascii_perturbation/documentation/perturbation.txt:1]: It's the ascii file needed for a simulation if the use didn't run the script

**Pull request closed without merging**